### ISSUE TITLE
Computing train embeddings

### ISSUE THREAD

**Issue by estanley16** [guidelabs/infembed#8]
Hi there, 
I was able to successfully use the tutorial in `imagenet_case_study_fast_kfac.ipynb` on my own data to compute influence embeddings for my test set. 

However, I would like to find slice opponents, so I am wondering how to compute influence embeddings for the **training** data? 
I tried calling `predict` for my training data on the embedder that was `fit` to the training data, but the results seemed wrong as all the embeddings were essentially zero.
Alternatively, is there a simpler and/or built-in way to compute slice opponents?

thanks!

**Comment by adebayoj** [comment]
Thanks for sharing the update! Indeed, these embeddings should not be zero. Right now, we don't have a direct way to compute slice opponents, but we can include one in a revamp of the repo. First, we'll reproduce the imagenet case study that you just found, and get back to you. 

Thanks!
